kolibri/plugins/management/assets/src/actions.js

```diff
@@ -24,23 +24,31 @@ const samePageCheckGenerator = require('kolibri.coreVue.vuex.actions').samePageC
  * The methods below help map data from
  * the API to state in the Vuex store
  */
-
-function _userState(data) {
-  // assume just one role for now
-  let kind = UserKinds.LEARNER;
-  if (data.roles.length && data.roles[0].kind === 'admin') {
-    kind = UserKinds.ADMIN;
+function _topUserKind(rolesObjectArray) {
```

Comment:
Also, just realized that the `SUPERUSER` check in `user-edit-modal` won't ever be triggered when we collapse it to an `ADMIN`. We probably do have to keep this info somehow, but I'm curious as to how we're representing it.